pkg/kv/migration.go

```diff
@@ -39,21 +39,20 @@ func (d *DatabaseMigrator) Migrate(ctx context.Context) error {
 	return nil
 }
 
-func ValidateSchemaVersion(ctx context.Context, store Store, migrationRequired bool) error {
+var errMigrationRequired = errors.New("wrong kv version")
+
+func ValidateSchemaVersion(ctx context.Context, store Store) error {
 	kvVersion, err := GetDBSchemaVersion(ctx, store)
-	if errors.Is(err, ErrNotFound) && !migrationRequired {
-		logging.Default().Info("No KV Schema version, setup required")
-		return nil
+	if errors.Is(err, ErrNotFound) {
+		// probably a new installation
+		return ErrNotFound
```

Comment:
Think we can drop this one - we just slice the information in the err to just ErrNotFound

Reply:
Not sure I follow - do you suggest I return nil here?